functions/list.go

```diff
@@ -16,36 +16,46 @@ import (
 // List - list network details for specified networks
 // long flag passed passed to cmd line will list additional details about network including peers
 func List(net string, long bool) {
+	output := make([]map[string]any, 0)
 	found := false
 	nodes := config.GetNodes()
 	for network := range nodes {
 		if network == net || net == "" {
 			found = true
 			node := nodes[network]
-			connected := "Not Connected"
-			if node.Connected {
-				connected = "Connected"
+			entry := map[string]any{
+				"network":      node.Network,
+				"connected":    node.Connected,
+				"id":           node.ID,
+				"ipv4_address": node.Address.String(),
+				"ipv6_address": node.Address6.String(),
 			}
-			fmt.Println()
-			fmt.Println(node.Network, connected, node.ID, node.Address.String(), node.Address6.String())
 			if long {
 				peers, err := getNodePeers(node)
 				if err != nil {
 					logger.Log(1, "failed to get peers for node: ", node.ID.String(), " Err: ", err.Error())
 					continue
 				}
-				fmt.Println("  Peers:")
+				entry["peers"] = make([]map[string]any, 0)
 				for _, peer := range peers {
-					fmt.Println("    ", peer.PublicKey, peer.Endpoint, "\n    AllowedIPs:")
+					p := map[string]any{
+						"public_key":  peer.PublicKey,
+						"endpoint":    peer.Endpoint,
+						"allowed_ips": make([]string, 0),
+					}
 					for _, cidr := range peer.AllowedIPs {
-						fmt.Println("    ", cidr.String())
+						p["allowed_ips"] = append(p["allowed_ips"].([]string), cidr.String())
 					}
+					entry["peers"] = append(entry["peers"].([]map[string]any), p)
 				}
 			}
+			output = append(output, entry)
 		}
 	}
 	if !found {
 		fmt.Println("\nno such network")
```

Comment:
"github.com/kr/pretty"
currently, we use this package to pretty print, don't need to create a function